File: domain/src/main/java/com/arturogutierrez/openticator/executor/PostExecutionThread.java
package com.arturogutierrez.openticator.executor;

import rx.Scheduler;

/**
 * Executor where the response of the work done by interactors is delivered, it will be done on
 * UI thread to allow modify the UI
 */
public interface PostExecutionThread {

  Scheduler getScheduler();
}

File: domain/src/main/kotlin/com/arturogutierrez/openticator/executor/PostExecutionThread.kt
package com.arturogutierrez.openticator.executor

import io.reactivex.Scheduler

/**
 * Executor where the response of the work done by interactors is delivered, it will be done on
 * UI thread to allow modify the UI
 */
interface PostExecutionThread {
  val scheduler: Scheduler
}
